sdg/DocumentationService.py

```diff
@@ -0,0 +1,159 @@
+import os
+import sdg
+import numpy
+from slugify import slugify
+
+class DocumentationService:
+    """HTML generation to document outputs built with this library.
+
+    Note that this is meant to document particular builds, not the library in
+    general. The idea is that each time this library is used to build/convert
+    some SDG-related data, this class can be used to generate human-friendly
+    HTML pages documenting the specifics of the build (such as endpoint URLs).
+    """
+
+
+    def __init__(self, outputs, folder='_site', branding='Build docs', languages=None, intro=''):
```

Comment:
Wouldn't the documentation generation require at least one language to work? If so (I might be missing something), you might want to inject this dependency on your constructor (omit `=None`), or you could default it to `languages=['en']`

Reply:
No actually it should be able to work without a language. In sdg-build we have this concept of "translated builds", where you specify one or more languages, and if you do, it creates entire builds for each of those languages, in subfolders. But this is not required.